bayesian/default_config.py

```diff
@@ -48,7 +48,8 @@
 
 SNYK_API_TOKEN_VALIDATION_URL = os.getenv('SNYK_API_TOKEN_VALIDATION_URL',
                                           'https://snyk.io/api/v1/verify/token')
-ENCRYPTION_KEY_FOR_SNYK_TOKEN = os.getenv('ENCRYPTION_KEY_FOR_SNYK_TOKEN', 'SNYK')
+ENCRYPTION_KEY_FOR_SNYK_TOKEN = os.getenv('ENCRYPTION_KEY_FOR_SNYK_TOKEN',
```

Comment:
`user[PUT]` API is not using this value here but one in https://github.com/fabric8-analytics/fabric8-analytics-utils/blob/e2207358ecb8289e2e941b0011d8b86c848e8a21/f8a_utils/default_config.py#L6

Reply:
Although it's good to have this change here too.

Reply:
Done